config/default.js

```diff
@@ -22,6 +22,9 @@ module.exports = {
       save: 'save',
       new: 'Save Revision',
     },
+    plugins: {
+      directory: [path.join(__dirname, '../input-plugins')],
```

Comment:
We need another way to tell content-types to grab all the input-plugin from the host `punchcard`. In the future, we might ship more plugins with punchcard.